sdk/servicebus/service-bus/src/receivers/receiver.ts

```diff
@@ -611,38 +615,39 @@ export class ServiceBusReceiverImpl implements ServiceBusReceiver {
     };
   }
 
-  completeMessage(message: ServiceBusReceivedMessage): Promise<void> {
-    const msgImpl = message as ServiceBusMessageImpl;
-    return msgImpl.complete();
+  async completeMessage(message: ServiceBusReceivedMessage): Promise<void> {
```

Comment:
I had intentionally not used the `async` here when adding these methods as we are not really awaiting on anything inside